src/github.com/getlantern/peerscanner/web.go

```diff
@@ -51,36 +56,46 @@ func startHttp() {
 // register is the entry point for peers registering themselves with the service.
 // If peers are successfully vetted, they'll be added to the DNS round robin.
 func register(resp http.ResponseWriter, req *http.Request) {
-	name, ip, port, err := getHostInfo(req)
-	if err == nil && port != 443 {
-		err = fmt.Errorf("Port %d not supported, only port 443 is supported", port)
+	name, ip, port, supportedFronts, err := getHostInfo(req)
+	if err == nil && !(port == "80" || port == "443") {
+		err = fmt.Errorf("Port %d not supported, only ports 80 and 443 are supported", port)
```

Comment:
But if you haven't started I'll give it a shot.
O 17 Mar, 2015 06:45, "myleshorton" <notifications@github.com> escribiu:

> In src/github.com/getlantern/peerscanner/web.go
> <https://github.com/getlantern/flashlight-build/pull/129#discussion_r26550753>
> :
>
> > @@ -51,36 +56,46 @@ func startHttp() {
> >  // register is the entry point for peers registering themselves with the service.
> >  // If peers are successfully vetted, they'll be added to the DNS round robin.
> >  func register(resp http.ResponseWriter, req *http.Request) {
> > -	name, ip, port, err := getHostInfo(req)
> > -	if err == nil && port != 443 {
> > -		err = fmt.Errorf("Port %d not supported, only port 443 is supported", port)
> > +	name, ip, port, supportedFronts, err := getHostInfo(req)
> > +	if err == nil && !(port == "80" || port == "443") {
> > +		err = fmt.Errorf("Port %d not supported, only ports 80 and 443 are supported", port)
>
> I think the direct IP proxies are critical path at this point yeah.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/flashlight-build/pull/129/files#r26550753>.
>